Makefile

```diff
@@ -27,4 +27,10 @@ srpm:   dist build-pkgs-dist
 rpm:    srpm
 	mock -r epel-7-x86_64 rebuild $(NAME)-$(VERSION)-$(RELEASE).el7.src.rpm --resultdir=. --define "dist .el7"
 
+update-release:
+	sed -i tendrl-dashboard.spec \
+	  -e "/^Release:/cRelease: $(shell date +"%m_%d_%Y_%H_%M_%S")"
```

Comment:
Please see this comment related to the version format: https://github.com/Tendrl/api/issues/209#issuecomment-314371093.